docs/RELEASE.md

```diff
@@ -0,0 +1,46 @@
+# Release cycle and versioning
+
+## Versioning
+Versioning is using SemVer (X.Y.Z):
+- The X is a **major version** and is incremented when:
+    - Support for older Zabbix versions is removed.
+    - Support for older Ansible versions is removed.
+    - Support for older Python versions is removed.
+    - Modules, roles or plugins are removed.
+    - Module or role functionality is removed.
+    - Other breaking changes or backward-incompatible changes are introduced.
+
+- The Y is a **minor version** and is incremented when:
+    - Support for new Zabbix versions is added.
+    - Support for new Ansible versions is added.
+    - Support for new Python versions is added.
+    - A new module, role, plugin, etc., is added.
+    - New features are introduced to modules, roles, plugins.
+    - A functionality of components is adjusted in a backward-compatible way.
+
+- The Z is a **patch version** and is incremented when:
+    - Bugs are fixed in a backward-compatible way.
+    - Documentation fixes and smaller changes are introduced.
+## Releases
+Release dates are not fixed. Instead, they will be discussed at the beginning of each month following this guideline:
+- The version increment will depend on the content that will be included in the release, as discussed in the *Versioning* section.
+- New collection releases may be a result of this discussion if necessary.
+- There may be several releases during the month if needed.
+## Collection support
+The latest release of the community.zabbix is always supported.
+Older releases, which are included in the still supported Ansible versions, may obtain occasional backports or bug fixes when necessary. [1]
```

Comment:
This, unfortunately, we need to do, since we are going to be a part of ansible releases.